### ISSUE TITLE
OmniCC options break if WeakAuras is enabled

### ISSUE THREAD

**Issue by Nercus** [tullamods/OmniCC#343]
**Give us a clear and concise description of the problem, and what you expect to happen**

If WeakAuras is enabled the OmniCC options break:
- Rules and Profiles tab are missing
- Options only open when rightclicking to OmniCC tab

_Tested with only WeakAuras and OmniCC enabled!_

**Do you have any steps to reproduce this issue?**

1. Start with only WeakAuras and OmniCC enabled.
2. Go to Interface OmniCC options.

**Are you getting any error messages?**

3 Errors happen when OmniCC options tab gets clicked:

```
1x OmniCC\config\rules.lua:34: attempt to concatenate field 'id' (a nil value)
[string "@OmniCC\config\rules.lua"]:34: in function <OmniCC\config\rules.lua:33>
[string "@OmniCC\config\rules.lua"]:166: in function `RefreshRuleOptions'
[string "@OmniCC\config\rules.lua"]:170: in main chunk
[string "=[C]"]: ?
[string "=[C]"]: in function `LoadAddOn'
[string "@OmniCC\main\main.lua"]:35: in function <OmniCC\main\main.lua:33>
[string "=[C]"]: in function `Show'
[string "@FrameXML\InterfaceOptionsFrame.lua"]:29: in function `InterfaceOptionsList_DisplayPanel'
[string "@FrameXML\InterfaceOptionsFrame.lua"]:47: in function `InterfaceOptionsListButton_OnClick'
[string "*:OnClick"]:2: in function <[string "*:OnClick"]:1>

Locals:
Skipped (In Encounter)
```

```
1x OmniCC\config\options-Options.lua:42: attempt to index field 'rules' (a nil value)
[string "@OmniCC\config\options-Options.lua"]:42: in main chunk
[string "=[C]"]: ?
[string "=[C]"]: in function `LoadAddOn'
[string "@OmniCC\main\main.lua"]:35: in function <OmniCC\main\main.lua:33>
[string "=[C]"]: in function `Show'
[string "@FrameXML\InterfaceOptionsFrame.lua"]:29: in function `InterfaceOptionsList_DisplayPanel'
[string "@FrameXML\InterfaceOptionsFrame.lua"]:47: in function `InterfaceOptionsListButton_OnClick'
[string "*:OnClick"]:2: in function <[string "*:OnClick"]:1>

Locals:
Skipped (In Encounter)
```

```
8x (null)
```

**What version of OmniCC are you running?**

OmniCC Version: `9.0.1`
WeakAuras Version: `3.0.5`

**What version of World of Warcraft are you on?**

```lua
[1]="9.0.1", 
[2]="36372", 
[3]="Oct 23 2020", 
[4]=90001
```

**What other addons are you running?**

_Tested with only WeakAuras and OmniCC enabled!_

**Comment by Tuller** [comment]
This should be resolved

**Comment by Nercus** [comment]
Can confirm that the issue is resolved in OmniCC version 9.0.2.
